aiida/orm/utils/remote.py

```diff
@@ -69,7 +71,16 @@ def get_calcjob_remote_paths(pks=None, past_days=None, older_than=None, computer
         filters_calc['mtime'] = {'>': timezone.now() - timedelta(days=past_days)}
 
     if older_than is not None:
-        filters_calc['mtime'] = {'<': timezone.now() - timedelta(days=older_than)}
+        older_filter = {'<': timezone.now() - timedelta(days=older_than)}
+        # Check if we need to apply the AND condition
+        if 'mtime' not in filters_calc:
+            filters_calc['mtime'] = older_filter
```

Comment:
I think these two time filters need to be used separately, otherwise, how do you know we should use 'AND' or 'OR' when they both applied? 
I would suggest just adding a check and raising a CLI exception here said these two options can not be applied at the same time.

Reply:
I think `and` is what makes most sense? Similar to other commandline tools like `find -mtime \+1 -mtime -10`.

Reply:
> Similar to other commandline tools like find -mtime \+1 -mtime -10

Cool! Good to know this. Then I would say I agree with using as `AND` because it is more system compatible.